### ISSUE TITLE
Other languages

### ISSUE THREAD

**Issue by cesagonchu** [XoopsModules25x/tdmdownloads#87]
Can you remove french language from here ?

https://github.com/XoopsModules25x/tdmdownloads/tree/master/language

It's not up to date.

Thanks!

**Comment by mambax7** [comment]
The languages should have now the same constants, but some will need a French translation. Do you still want me to delete the French translation?

**Comment by cesagonchu** [comment]
Yes, I always thought it's better to leave only the English language in a repository, it avoids bug problems.
If you still want the latest French translation, I have proposed it here: 
https://www.frxoops.org/modules/newbb/viewtopic.php?post_id=242648#forumpost242648
